src/main/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacade.java

```diff
@@ -1561,14 +1558,13 @@ public final Response getEventOverviews(@Context final HttpServletRequest reques
                     eventOverviewBuilder.put("location", event.getLocation());
                 }
 
-                eventOverviewBuilder.put("numberOfConfirmedBookings",
-                        this.bookingManager.countNumberOfBookingsWithStatus(event.getId(), BookingStatus.CONFIRMED));
-                eventOverviewBuilder.put("numberOfWaitingListBookings",
-                        this.bookingManager.countNumberOfBookingsWithStatus(event.getId(), BookingStatus.WAITING_LIST));
-                eventOverviewBuilder.put("numberAttended",
-                        this.bookingManager.countNumberOfBookingsWithStatus(event.getId(), BookingStatus.ATTENDED));
-                eventOverviewBuilder.put("numberAbsent",
-                        this.bookingManager.countNumberOfBookingsWithStatus(event.getId(), BookingStatus.ABSENT));
```

Comment:
Had a look at this in my IDE and it suggests this method isn't being used now. Double-check that but if so you may want to clean it out.

Reply:
Its used when someone uses /overview

Reply:
... I'm pointing at the removed reference to `countNumberOfBookingsWithStatus`, not `getEventOverviews`